packages/react-frontend/src/routes/FooterFiles/AboutUs.jsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from "prop-types";
+import Navbar from "../Navbar";
+//import "../../index.css";
+import Footer from "./Footer";
+import "./Footer.css"
+
+
+
+export default function About() {
+
+  return (
+    <>
+    <div className="loading-text">
+      <Navbar withLogo={true} />
+      <span className="about-us">
+      <h2>About Us: Our Story</h2>
+      </span>
+      <span className="about-us">
+      
```

Comment:
can i suggest adding a little bit more space between the "About Us: Our Story" title  and the rest of the text ? I think overall it looks great though!